apps/utils/draft_pull_request.py

```diff
@@ -224,19 +213,6 @@ def cli(
         log.error(f"Failed to create draft pull request:\n{response.json()}")
         return
 
-    # modify the PR body to include the diff link
-    if step_update:
-        diff_link = f"{js['html_url']}/files/{repo.head.commit.hexsha}..HEAD"
```

Comment:
I think this was quite useful, I'm sad to see this go!

Reply:
I thought that too, but now I think it's more intuitive to just copy the link from the docs or select correct commits if you're a reviewer. I'm sure we'll find a better way to display this in the future.